priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs

```diff
@@ -0,0 +1,84 @@
+defmodule HexWeb.Repo.Migrations.RemovePackageDownloadsPk do
+  use Ecto.Migration
+
+  def up do
```

Comment:
Maybe it's not worth it just to drop the primary key from the view or is there some specific reason?